README.md

```diff
@@ -190,3 +190,16 @@ Use `mockery -print` to have the resulting code printed out instead of written t
 
 When your interfaces are in the main package you should supply the `-inpkg` flag.
 This will generate mocks in the same package as the target code avoiding import issues.
+
+### Extracting Function Names ###
+
+When using mocks, it's recommended to extract the function names using the below utility function in order to make your unit-tests more robust
+for code changes.
+
+```go
+import "github.com/stretchr/testify/utils"
+
+Mock.On(utils.GetFunctionName(mock.Function), mock.Anything).Return(func(s string) string {
```

Comment:
Actually, I don't understand how this would make anything easier. If you end up changing the name of the function, you would have to modify your `Mock.On` call anyway, because the function name changed. Plus it looks like this `utils` package no longer exists.

Reply:
1. After changing the function name, it will be catched during compilation which is better than during running the tests (and then figure out the function name have changed).
2. Changing the function name using IDE will automatically change this instance as well in contrary to string.

Reply:
That's a very good point. If you can fix the issue with testify/utils no longer existing (possibly find some suitable replacement), we can get this merged.